src/Components/main.js

```diff
@@ -15,6 +15,7 @@ import Container from '@material-ui/core/Container';
 import Grid from '@material-ui/core/Grid';
 import Divider from '@material-ui/core/Divider';
 import "../styles/Main.css";
+import Alert from '@material-ui/lab/Alert';
```

Comment:
What's this for?

Reply:
That is a typo :D Looks like vscode pulled in 'Alert' when I was typing 'AlertBox'

Reply:
Ohhh ok :D
Also, weren't we going to have the alerts as a banner or something? (and red in color)